dll/opengl/opengl32/swimpl.c

```diff
@@ -651,9 +651,9 @@ static inline COLORREF MAKE_COLORREF_24(const struct pixel_format *format, ULONG
     if (format->iPixelType == PFD_TYPE_COLORINDEX)
         return PALETTEINDEX(Color);
 
-    R = (Color & 0xFF0000) >> 16;
```

Comment:
What about using the existing macros `GetRValue`, `GetGValue`, `GetBValue` that would also allow the code to be self-descriptive and would have avoided this bug on the first place? (and make the code more portable in case the RGB encoding could change in different endian machines)

Reply:
it should be better now